client/src/components/Onboarding/OnboardingUser.scss

```diff
@@ -0,0 +1,42 @@
+.onboarding {
+  padding: 1em 1em 1em 1em;
+}
+
+.onboarding__access {
+  text-align: right;
+}
+
+.onboarding__title {
+  font-size: 1.5em;
+  line-height: 2em;
+  width: 100%;
+  border-bottom: 1px solid black;
+  margin-bottom: 1em;
+}
+
+.onboarding-actions,
+.onboarding-processing {
+  .loading-icon {
+    display: inline-block;
+    width: inherit;
+  }
```

Comment:
__Hmm…__ The need for this suggests that the styling of a `<Section placement="inline">` is broken (I've recently suspected this), or needs to handle another use case.

Reply:
Resolved using a flexbox for now.